app/src/main/java/com/mashup/lastgarden/ui/scent/ScentViewPagerAdapter.kt

```diff
@@ -27,30 +28,32 @@ class ScentViewPagerAdapter(
     }
 
     override fun onBindViewHolder(viewHolder: ScentViewHolder, position: Int) {
-        viewHolder.bind(list[position])
+        list?.get(position)?.let { viewHolder.bind(it) }
     }
 
     override fun getItemCount(): Int {
-        return list.size
+        return list?.size ?: 0
     }
 
-    private fun ScentViewHolder.bind(item: ScentItem) {
-        glideRequests.load(item.imgUrl).into(binding.scentImageView)
-        val pageCount = (bindingAdapterPosition + 1).toString() + "/" + list.size
+    private fun ScentViewHolder.bind(item: Story) {
+        glideRequests.load(item.perfumeImageUrl).into(binding.scentImageView)
         binding.run {
+            val pageCount =
+                (bindingAdapterPosition + 1).toString() + " / " + list?.size?.let {
+                    numberFormatter(it.toLong())
+                }
             pageCountTextView.text = pageCount
-            profileImageView.setImageUrl(glideRequests, item.imgProfile)
-            nicknameTextView.text = item.nickname
-            dateTextView.text = item.date
-            tagListTextView.text = TextUtils.join(" ", item.tagList)
-            commentCountTextView.text = item.commentCount.toString()
-            likeCountTextView.text = item.likeCount.toString()
-            commentImageView.setOnClickListener { listener?.onCommentClick(item.scentId) }
-            likeImageView.setOnClickListener { listener?.onLikeClick(item.scentId) }
-            likeImageView.loadImage(
-                glideRequests,
-                if (item.likeState) R.drawable.ic_like else R.drawable.ic_dislike
-            )
+            profileImageView.setImageUrl(glideRequests, item.perfumeImageUrl)
+            nicknameTextView.text = item.userNickname
+            dateTextView.text = dateConverter(item.createdAt)
+            var tags = ""
+            item.tags?.forEach { tags += it.contents + " " }
+            tagListTextView.text = tags
```

Comment:
```suggestion
            tagListTextView.text = item.tags.map { it.contents + " "}
```
이렇게 하면 불필요한 변수선언을 없앨 수 있습니다